model.py

```diff
@@ -1,8 +1,8 @@
 import pandas as pd

```

Comment:
GPT summary of [6b369b](https://github.com/SampatSuthar/ml-demo/blob/f3fc2c559b5eac826cd89d51e16fef43ebaff446/model.py#6b369b7e2aa5004d39fa1ad8d57c3cc72631606e) - [930c68](https://github.com/SampatSuthar/ml-demo/blob/4e14c9160140ec22e9e0c5878ed9eed70b74c263/model.py#930c68f8017150549757dff223c4b2e3dbd89f2a):
* Imports for pandas, numpy, matplotlib, and seaborn were added
* The import for pickle was commented out